components/agentList/index.jsx

```diff
@@ -0,0 +1,250 @@
+/**
+ * Copyright 2020 Inrupt Inc.
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a copy
+ * of this software and associated documentation files (the "Software"), to deal in
+ * the Software without restriction, including without limitation the rights to use,
+ * copy, modify, merge, publish, distribute, sublicense, and/or sell copies of the
+ * Software, and to permit persons to whom the Software is furnished to do so,
+ * subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED,
+ * INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A
+ * PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT
+ * HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION
+ * OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+ */
+
+import React, { useContext, useEffect, useState } from "react";
+import PropTypes from "prop-types";
+import clsx from "clsx";
+import { createStyles } from "@material-ui/core";
+import { makeStyles } from "@material-ui/styles";
+import { useBem } from "@solid/lit-prism-patterns";
+import {
+  DrawerContainer,
+  Table as PrismTable,
+} from "@inrupt/prism-react-components";
+import {
+  addStringNoLocale,
+  addUrl,
+  createThing,
+  getSourceUrl,
+  getStringNoLocale,
+  getThing,
+  getUrl,
+} from "@inrupt/solid-client";
+import { Table, TableColumn, useSession } from "@inrupt/solid-ui-react";
+import { vcard, foaf } from "rdf-namespaces";
+import SortedTableCarat from "../sortedTableCarat";
+import Spinner from "../spinner";
+import AgentAvatar from "../agentAvatar";
+import styles from "./styles";
+
+import { useRedirectIfLoggedOut } from "../../src/effects/auth";
+import useAddressBookOld from "../../src/hooks/useAddressBookOld";
+import useContactsOld from "../../src/hooks/useContactsOld";
+import { chain } from "../../src/solidClientHelpers/utils";
+import useProfiles from "../../src/hooks/useProfiles";
+import ProfileLink from "../profileLink";
+import SearchContext from "../../src/contexts/searchContext";
+import { deleteContact, vcardExtras } from "../../src/addressBook";
+import ContactsDrawer from "../contactsList/contactsDrawer";
+import ContactsEmptyState from "../contactsList/contactsEmptyState";
+
+const useStyles = makeStyles((theme) => createStyles(styles(theme)));
+
+// termporarily using mock data for apps for dev purposes until we have audit list
```

Comment:
```suggestion
// FIXME: temporarily use mock data for apps for dev purposes until we have audit list
```

"FIXME" makes searching the codebase for things we forgot easier.